requirements-tests.txt

```diff
@@ -3,4 +3,4 @@ pytest-cov
 pytest-mock
 flaky
 pytest-benchmark
-git+https://github.com/PennyLaneAI/pennylane.git@master
\ No newline at end of file
+git+https://github.com/PennyLaneAI/pennylane.git@v0.35.1-rc
```

Comment:
Neat @AmintorDusko .

Reply:
Yes, this is a new file. We can improve on that.

Reply:
> hmm.. interesting that we need to change this in multiple places! we can add the files with this required change to the release guideline to have them checked/updated for each release. @AmintorDusko what do you think?

Done, I've made a couple small updates.